ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,13 @@
+## Feature Request
+
+### Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is. Ex. I have an issue when [...]
+
+### Describe the solution you'd like
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+### Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+### Does this feature provide immediate value to the end customer
+Is this feature absolutely necessary? Will it benifit end users? Is this some sort of snake-oil mascarading as being the latest tech trend? Will we be coding outselves into a corner? Is this a waste of time? Are we locking ourselves into a provider, framework, service, product, subscription of which we will spend equal if not more time getting ourselves out of?
```

Comment:
Some of these questions are rather not connected with this section, ie.:
- Are we locking ourselves into a provider, ...
- Is this a waste of time (like question "are you a terrorist?" on US and A visa form...)
How do they correspond to main question `Does this feature provide immediate value to the end customer`?
Just don't see a point to put it in here, maybe another section?

Reply:
Add an impact - Level 1,2 or 3 where:
1- Stopping BuG - need immediate action (daily)
2- Important - to be handled ASAP (week or two)
3- Minor - Can be handled (release or later)